README.md

```diff
@@ -43,20 +46,21 @@ Note that if you are not on Debian based system you will need to follow https://
 Follow the instructions in your terminal to open the aplication in your browser.
 
 
-
 Upload Data
 ------------
 
-In order to upload some data use the dataload/import_to_elesticsearch.py command line tool e.g:
-    python dataload/import_to_elesticsearch.py --clean filename1.csv filename2.csv *.json
+In order to upload some data use the dataload/import_to_elasticsearch.py command line tool e.g:
+    python dataload/import_to_elasticsearch.py --clean filename1.csv filename2.csv *.json
 
-The clean command is optional it will delete the index and start again, so leave it off if you want to add just another file to an existing index.
+The clean command is optional; it will delete the index and start again, so leave it off if you want to add just another file to an existing index.
 You can specify as many file or patterns as you like at the end of the command.
 
 ### Getting data for upload
 
 There is a list of 360Giving datasets at http://www.threesixtygiving.org/data/find-data/. There's an API for this list http://data.threesixtygiving.org/data.json and some code to help download from it -  https://github.com/ThreeSixtyGiving/datagetter
 
+If your data is in a flat format (eg Excel spreadsheet, CSV), or needs validating, you can use [CoVE](http://cove.opendataservices.coop/360/) to convert and validate your data.
```

Comment:
I think the current grantnav import tool will also automagically run flatten-tool for you. (Although maybe we don't want to commit to keeping this, as we don't use it ourselves any more.